### DISCUSSION TITLE
why does the transcribed result show mixed language even though I fix the language ?

### DISCUSSION THREAD

**Discussion by hynkang** [openai/whisper#1625] [Q&A]
I used language options to fix the language as the code line below 

> result = model.transcribe(audio, language='ko')        

But the result shows mixed language, as..

Input text: 안녕하세요 아아 
output transcription: 안녕하세요	ああ

Is it the problem of the model size? 
In other words, if I choose bigger model would it be solved?
I used 'base' model.

**Comment by qaixerabbas** [comment]
@hynkang you should try bigger model. I am not sure in your case but specifying language in transcribe function solves most issues with transcription.

**Comment by coastal45** [comment]
Is there a correspondence between 아아 and ああ, either phonetic or by definition? Sorry I don't know any Korean.
I have rarely noticed in Japanese transcription a line that seems to fall into translation mode unexpectedly.

**Reply by ryanheise** [reply]
Just to comment on the phonetics, yes they correspond to each other phonetically.

**Comment by qaixerabbas** [comment]
@coastal45 I am sorry but I have no knowledge about Korean or Japanese. Maybe you should contact someone who have worked with Korean in whisper.
